enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/PresenceCheckService.java

```diff
@@ -243,14 +260,43 @@ public Image selectPhotoForPresenceCheck(OwnerId ownerId, List<DocumentVerificat
         return identityVerificationService.getPhotoById(photoId);
     }
 
+    private void evaluatePresenceCheckResult(OwnerId ownerId,
+                                             IdentityVerificationEntity idVerification,
+                                             PresenceCheckResult result) {
+        switch (result.getStatus()) {
+            case ACCEPTED:
+                idVerification.setStatus(IdentityVerificationStatus.ACCEPTED);
+                idVerification.setTimestampLastUpdated(ownerId.getTimestamp());
+                logger.info("Presence check accepted, {}", ownerId);
+                break;
+            case FAILED:
+                idVerification.setErrorDetail(result.getErrorDetail());
+                idVerification.setStatus(IdentityVerificationStatus.FAILED);
+                idVerification.setTimestampLastUpdated(ownerId.getTimestamp());
+                logger.warn("Presence check failed, {}, errorDetail: '{}'", ownerId, result.getErrorDetail());
+                break;
+            case IN_PROGRESS:
+                logger.debug("Presence check still in progress, {}", ownerId);
+                break;
+            case REJECTED:
+                idVerification.setRejectReason(result.getRejectReason());
+                idVerification.setStatus(IdentityVerificationStatus.REJECTED);
+                idVerification.setTimestampLastUpdated(ownerId.getTimestamp());
+                logger.warn("Presence check rejected, {}, rejectReason: '{}'", ownerId, result.getRejectReason());
```

Comment:
I am not sure whether we should use a `warn` level given how frequent this error is. I would prefer using `info` level.